3.10/arangosync-migration-tool.md

```diff
@@ -0,0 +1,232 @@
+- Safety comes first - pre-checks and potential failures are carefully handled.
+- Your data is secure and fully encrypted.
+- Ease-of-use with a live migration while your local cluster is still in use.
+- Get access to what a cloud-based fully managed service has to offer: 
+high availability and reliability, elastic scalability, and much more.
+
+## Prerequisites 
+
+Before getting started, make sure the following prerequisites are in place:
+
+- Go to [Oasis Cloud](https://cloud.arangodb.com/home){:target="_blank"}
+and sign in. If you don’t have an account yet, sign-up to create one.
+
+- Generate an Oasis API key and API secret. See a detailed guide on 
+[how to create an API key](oasis/api-getting-started.html#creating-an-api-key).
+
+### Setting up the target deployment in Oasis
+
+Continue by [creating a new Oasis deployment](oasis/deployments.html#how-to-create-a-new-deployment)
+or choose an existing one.
+
+The target deployment in Oasis requires specific configuration rules to be
+set up before the migration can start:
+
+- **Configuration settings**: The target deployment must be compatible with the
+source data cluster. This includes the ArangoDB version that is being used,
+the DB-Servers count, and disk space.
+- **Deployment region and cloud provider**: Choose the closest region to your
+data cluster. This factor can speed up your migration to the cloud.
+
+After setting up your Oasis deployment, wait for a few minutes to become 
+fully operational.
+
+## Running the migration tool
+
+The `arangosync-migration` tool provides a set of commands that allow you to:
+-  start the migration process
+-  check whether your source and target clusters are fully compatible
+-  get the current status of the migration process
+-  stop or abort the migration process
+-  switch the local cluster to read-only mode
+
+### Starting the migration process
+
+To start the migration process, run the following command:
+
+```bash
+arangosync-migration start
+```
+
+An optional `--check-only` option can be provided. When specified, you can check
+if your local cluster and target deployment are compatible and if the migration 
+can start. No data is sent to Oasis.
+
+Before the actual migration starts, the `start` command also runs some
+pre-checks. Among other things, it measures the disk space which is occupied
+by your ArangoDB cluster. If you are using the same data volume for ArangoDB
+servers and other data as well, the measurements can be incorrect.
+Provide the `--source.ignore-metrics` option to overcome this.
+
+Once the migration starts, the local cluster enters into monitoring mode and the
+synchronization status is displayed in real-time. If you don't want to see the
+status you can also terminate this process, as the underlying agent process
+continues to work. If something goes wrong, restarting the same command restores
+the replication state.
+
+To restart the migration, use the `stop --abort` command.
```

Comment:
This is a bit incorrect: this command will not restart migration, it only stops it.

Suggestion:
```
To restart migration, first `stop` or `stop --abort` the migration, then start it again using `start` command.
```